Removed the code to import scenarios from connector

cs-swati-jinde opened the pull request:
Removed the code to import scenarios from connector

cs-swati-jinde committed:
Removed the code to import scenarios from connector

Removed the code to import scenarios from connector

cs-swati-jinde committed:
Updated release version in info.json

Updated release version in info.json

cs-tushar-kanade commented on fortisoar-soc-simulator/info.json:
```diff
@@ -9,16 +9,6 @@
     "category": "information",
     "icon_medium_name": "fortisoar_medium.png",
     "configuration": {
-        "fields": [
```
Syntax error here , Fields [  ...

cs-tushar-kanade replied:
check RAW file , we see ] in file

cs-swati-jinde replied:
Resolved, Please review

cs-tushar-kanade commented on fortisoar-soc-simulator/utils.py:
```diff
@@ -11,48 +11,6 @@
-
-    scenario_path=os.path.join(os.path.dirname(__file__), "scenarios/"+scenario+"/scenario_configuration.json")
-    if(os.path.exists(scenario_path)):
-        set_config_job(scenario_path,scenario_title=scenario)
```
apart from import_records there are supporting functions , please remove them as well .e.g set_config_job , get_config_job

cs-swati-jinde replied:
Resolved, removed get_config_job(),set_config_job,_upload_file,_create_import_job

cs-tushar-kanade reviewed (changes requested):
Please do changes are per comments

cs-swati-jinde committed:
Incorporated Review comments 1

Incorporated Review comments 1

cs-swati-jinde committed:
Merge branch 'develop' into release/2.0.0

cs-swati-jinde committed:
Removed the scenarios folder and updated info.json to remove import scenario checkbox

1.Removed the All scenarios and folder from connector
2.Updated info.json , remove import scenario checkbox

cs-tushar-kanade reviewed (approved):
Looks good

cs-tushar-kanade commented:
DO NOT MERGE YET , WE DO NOT WANT TO RELEASE CONNECTOR

cs-suyog-jinturkar commented on fortisoar-soc-simulator/utils.py:
```diff
@@ -11,83 +11,6 @@
-    job_id = response.get('@id').split("/")[-1]
-    return job_id
 
 def load_threat():
```
What is this function do?

cs-swati-jinde committed:
Merge branch 'develop' into release/2.0.0
